production.json

```diff
@@ -297,6 +297,11 @@
       }
     },
     "ethereum": {
+      "accountant": {
```

Comment:
should be not present, rather than null values

Reply:
yes, but JSON writing doesn't work otherwise and i don't want to block on the issue :(

Reply:
i can remove it from the checked-in config, but it is needed to run the script, so i figured it would make sense to check in the final input to the script